### DISCUSSION TITLE
Problem with .iso image error, and other questions.

### DISCUSSION THREAD

**Discussion by Indibilio** [erichelgeson/BlueSCSI-v1#209] [Q&A]
Hello, I am writing to you from Spain, sorry if my English is bad.

I'm using a BlueSCSI 1.1a board with 20221203-STM32F1-USB firmware, on a Performa 475.

I have loaded a pre-installed disk image from the internet with 7.5.3 in English. I want to install the operating system 7.5.3 in Catalan on a hard disk image, obtained from this link:

https://winworldpc.com/download/3ec5a014-c2a6-18c3-9a11-c3a4e284a2ef

I configure it as "CD00 - 7.5.3.iso" and the disk is mounted by booting from HD image, but when I run the installer it gives me a message similar to "cannot be executed due to disk error". I have tried adding 2048, 1024, 512 and even 256 after 00 (like CD00_2048...) and all of them the CD shows up on the desktop, but I get the same error.

Do you know how I can install this CD in the HD image?

With this firmware version "BlueSCSI SD Transfer" should work, right? Because it tells me that "not compatible BlueSCSI's found". The lo file show:
_In Main!
SCSISelect-id:7 err:4
SCSISelect-id:5 err:2_

Is it normal that when I move the mouse or press a key, the speaker emits a noise? similar to what a real hard drive would do. I don't know if it is a failure of the equipment or it is produced by BlueSCSI.

Thanks!

**Comment by erichelgeson** [comment]
Lots of questions here - hopefully i get them all:

Please try a known good CD from here: https://github.com/erichelgeson/BlueSCSI/wiki/CD-Image-Support#known-good-macintosh-bootable-images I'd reccomend the legacy recovery CD - it has many installers.

Also you could just grab our pre-made 7.5 image from the Usage page on the wiki.

Please use a blank drive made by Disk Jockey or that is on the Usage page as well.

BlueSCSI transfer utility as the readme states - requires you to flash the firmware from that repo. It is a beta and not in the main release yet. Once you do you'll be able to use it.

On the noise - sounds like some interference - these are 20-30 year old machines and likely need to be recapped. The BlueSCSI doesnt have activity when the mouse moves so I'd not think it's that.

**Reply by Indibilio** [reply]
Thank you very much for answering.

As I wanted to install 7.5.3 in Catalan, I finally did it in Basilisk II, I loaded the disk image and the HD image, it was mounted correctly, I installed it, and then I copied it in BlueSCSI.

Regarding the SD Transfer, if I add the BETA firmware, does it not alter the rest of the "stable" functions of the firmware? only add BETA support for SD Transfer?

Thank you for the advice of the Caps, I have ordered them and I will change them as soon as I receive them.

Thank you.

**Reply by erichelgeson** [reply]
> Regarding the SD Transfer, if I add the BETA firmware, does it not alter the rest of the "stable" functions of the firmware? only add BETA support for SD Transfer?

Flash it and give it a try, if you have any issues with the transfer app please let me know. You can always flash back to the latest too.
